docs/architecture.md

```diff
@@ -1,7 +1,7 @@
 # General architecture
 
-When running on Travis tests are executed in the several stages that are outlined below. 
-When it comes to local tests, steps are identical, but the push to the remote dockerhub repository is absent.
+When running on Travis tests are executed as specified in the file `.travis.yml` located in the root of the systemtests repository. The several stages of testing are outlined below.
+If running a local tests (compared to the standard build on the Travis servers) the steps are identical, but the push to the remote DockerHub repository is absent.
```

Comment:
Referring to the `travis.yml` file to make it clearer how tests are orchestrated.